exif2timestream.py

```diff
@@ -581,6 +689,11 @@ def main(opts):
     # beginneth the actual main loop
     start_time = time()
     cameras = parse_camera_config_csv(opts["-c"])
+    try:
+        global EXIF_DATE_MASK
+        EXIF_DATE_MASK = opts["-m"]
```

Comment:
I had it down as a try catch because I figured it was an optional argument. that is, that -m will only be defined when they want to use a custom mask to read in. 
If i do it with an if statement, and they dont put -m in the command line, it will error